runbotics/runbotics-orchestrator-ui/src/src-app/views/process/EditProcessDialog/EditProcessDialog.tsx

```diff
@@ -120,8 +120,13 @@ const EditProcessDialog: FC<EditProcessDialogProps> = ({
                 isDisabled: isLoading,
             }}
         >
-            <Box sx={isLoading && { pointerEvents: 'none', opacity: 0.7 }}>
-                <Content sx={{ overflowX: 'hidden' }}>
+            <Content sx={{ overflowX: 'hidden' }}>
+                <If condition={!isLoading} else={
+                    <Box sx={{ width: '500px'}}>
+                        <Loader/>
+                    </Box>
+                }
```

Comment:
I recommend using Show component from multi-condition library 😉 example in CustomCredentialSelectDialog.tsx (line 77)